mllib/src/main/scala/org/apache/spark/ml/util/Instrumentation.scala

```diff
@@ -148,12 +170,25 @@ private[spark] class Instrumentation[E <: Estimator[_]] private (
   }
 
 
+  // TODO: Remove this (possibly replace with logModel?)
   /**
    * Logs the successful completion of the training session.
    */
   def logSuccess(model: Model[_]): Unit = {
     log(s"training finished")
   }
+
+  def logSuccess(): Unit = {
+    log("training finished")
```

Comment:
We shouldn't have this `log` alias. I was wondering which log level it uses. Just use `logInfo` and remove `log(`.